controllers/operands/operandHandler.go

```diff
@@ -67,6 +66,12 @@ func NewOperandHandler(client client.Client, scheme *runtime.Scheme, isOpenshift
 		}...)
 	}
 
+	if ci.IsOpenshift() && ci.IsConsolePluginImageProvided() {
+		operands = append(operands, []Operand{
+			newConsoleHandler(client),
+		}...)
```

Comment:
the 2nd parameter does not have to be a slice.
```suggestion
		operands = append(operands, newConsoleHandler(client))
```